<!-- filepath: /workspaces/demo/装箱智多星/src/mainwindow.ui -->
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>装箱智多星</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="vehicleInfoTextEdit">
      <property name="placeholderText">
       <string>输入车辆信息...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="cargoInfoTextEdit">
      <property name="placeholderText">
       <string>输入货物信息...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="calculateButton">
      <property name="text">
       <string>计算装箱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="resultTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>装箱结果...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>